integr8ly/integreatly-operator
# MGDAPI-6055 - Update 3scale bundle (0.11.8-mas)

# Issue link
JIRA: [MGDAPI-6055](https://issues.redhat.com/browse/MGDAPI-6055)

# What
This PR bumps the 3scale-operator image used by RHOAM from `0.11.6-mas` to `0.11.8-mas`

# Verification steps
1. Provision a cluster
2. Verify fresh install scenario
3. Uninstall RHOAM
4. Verify upgrade scenario
